src/render/post/multipart/fixed/pre.js

```diff
@@ -0,0 +1,38 @@
+const text = require('../../../text')
+
+function generateMultipartFixedPre(params, boundary) {
+  return (
+    [
+      `formData = new FormData();`,
+      `formData.boundary = ${text(boundary)};`,
+      ...generateAppendData(params),
+    ].join('\n') + '\n'
+  )
+}
+
+function generateAppendData(params) {
+  return Array.from(params).reduce((acc, [name, valueSet]) => {
+    valueSet.forEach((value) => {
```

Comment:
Nit, would be nicer to use map instead of forEach :) 
```
Array.from(params).reduce((rows, [name, valueSet]) => [
  ...acc,
  Array.from(set).map((value) => {
    return line
  })
], [])

```

Reply:
yup, good catch